pallets/phala/src/compute/computation.rs

```diff
@@ -254,21 +255,21 @@ pub mod pallet {
 	#[pallet::getter(fn cool_down_period)]
 	pub type CoolDownPeriod<T> = StorageValue<_, u64, ValueQuery>;
 
-	/// The next id to assign to a mining session
+	/// The next id to assign to a computing session
 	#[pallet::storage]
 	pub type NextSessionId<T> = StorageValue<_, u32, ValueQuery>;
 
-	/// The block number when the mining starts. Used to calculate halving.
+	/// The block number when the computing starts. Used to calculate halving.
 	#[pallet::storage]
-	pub type MiningStartBlock<T: Config> = StorageValue<_, T::BlockNumber>;
+	pub type WorkingStartBlock<T: Config> = StorageValue<_, T::BlockNumber>;
```

Comment:
```suggestion
	pub type ComputeStartBlock<T: Config> = StorageValue<_, T::BlockNumber>;
```

Here "Mining" refer to the mining mechanism. I think it's better to replace it by "compute".